epos4.sh

```diff
@@ -0,0 +1,77 @@
+package: EPOS4
+version: "%(tag_basename)s"
+tag: "v4.0.0-alice1"
+source: https://github.com/alisw/EPOS4.git
+requires:
+  - ROOT
+  - fastjet
+env:
+  EPOVSN: "4.0.0"
+  EPO: "${EPOS_ROOT}/epos4/"
+  LIBDIR: "${EPO}bin"
+  OPT: "./"
+  HTO: "./"
+  CHK: "./"
```

Comment:
These will be set at build time for the dependencies of EPOS4. Is that what you really need?

Reply:
I removed them cause they aren't needed (for now).